Author: Alessio Treglia <quadrispro@ubuntu.com>
Author: Dennis Braun <d_braun@kabelmail.de>
Description: Disable -march-native and add a proper pkgconf variable to fix FTBFS.
Bug: http://bugs.debian.org/553630
Bug: http://bugs.debian.org/578278
Bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=950826
Last-Update: 2020-06-24

Index: aeolus/source/Makefile
===================================================================
--- aeolus.orig/source/Makefile
+++ aeolus/source/Makefile
@@ -22,11 +22,11 @@ SUFFIX := $(shell uname -m | sed -e 's/^
 PREFIX ?= /usr/local
 BINDIR ?= $(PREFIX)/bin
 LIBDIR ?= $(PREFIX)/lib$(SUFFIX)
+PKG_CONF ?= pkg-config
 
 VERSION = 0.10.4
 CPPFLAGS += -MMD -MP -DVERSION=\"$(VERSION)\" -DLIBDIR=\"$(LIBDIR)\"
 CXXFLAGS += -O2 -Wall
-CXXFLAGS += -march=native
 
 
 all:	aeolus aeolus_x11.so aeolus_txt.so
@@ -46,7 +46,7 @@ $(AEOLUS_O):
 XIFACE_O =	styles.o mainwin.o midiwin.o audiowin.o instrwin.o editwin.o \
 	midimatrix.o multislider.o functionwin.o xiface.o addsynth.o
 aeolus_x11.so:	CPPFLAGS += -D_REENTRANT
-aeolus_x11.so:	CPPFLAGS += $(shell pkg-config --cflags freetype2)
+aeolus_x11.so:	CPPFLAGS += $(shell $(PKG_CONF) --cflags freetype2)
 aeolus_x11.so:	CXXFLAGS += -shared -fPIC
 aeolus_x11.so:	LDFLAGS += -shared
 aeolus_x11.so:	LDLIBS += -lclthreads -lclxclient -lpthread -lXft -lX11
@@ -75,7 +75,6 @@ install:	aeolus aeolus_x11.so aeolus_txt
 	install -m 755 aeolus $(DESTDIR)$(BINDIR)
 	install -m 755 aeolus_x11.so $(DESTDIR)$(LIBDIR)
 	install -m 755 aeolus_txt.so $(DESTDIR)$(LIBDIR)
-	ldconfig $(PREFIX)/$(LIBDIR)
 
 
 clean:
